imports/ui/RouterComponents/activities/Activities.jsx

```diff
@@ -55,10 +55,14 @@ function Activities({ activitiesList, processesList, isLoading, history }) {
       return null;
     }
 
-    const futureProcesses = processesList.filter((process) =>
-      process.meetings.some((meeting) =>
-        moment(meeting.startDate).isAfter(yesterday)
-      )
+    const futureProcesses = processesList.filter((process) => {
+      if (process.meeting && process.meeting.some) {
```

Comment:
should be meetings, and if we had the meetings array initiated at createProcess, would probably be redundant no?

Reply:
yes at that point defaultValue wasn't set yet but generally there is nothing wrong checking nested keys step by step but we can delete if we want to